screens/home_screens/HotScreen.js

```diff
@@ -81,18 +82,68 @@ const hot_events = [
   
 
 export default class HotScreen extends React.Component {
+  state = {
+    posts:[ ]
+  }
+  componentDidMount(){
+      axios.get('https://quiet-spire-38612.herokuapp.com/api/events/5c607e3c8149f900160c89f6')
```

Comment:
this should be [https://quiet-spire-38612.herokuapp.com/api/events/](https://quiet-spire-38612.herokuapp.com/api/events/) if you want to get all events